code/Unity Project/Assembly-CSharp.csproj

```diff
@@ -1,5 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
```

Comment:
This is an attempt to borrow configuration from a project originally created using dotnet new command for .NET Core 2.1 then added Unity as a Nuget